node/rpc/src/lib.rs

```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 mod chain_api;
+mod wallet_api;
+
```

Comment:
Nit : Remove empty line

Reply:
why? I personally think it's a good thing to separate `mod` declarations from `use` imports

Reply:
Whoops didnt realize that it was convention

Reply:
it's no official convention, just opinionated, which is why I ask